runtime/fswatch.vim

```diff
@@ -0,0 +1,15 @@
+" autocommands for starting filesystem based file watcher
+
+if exists("did_fswatch_on")
+  finish
+endif
+let did_fswatch_on = 1
+
+echo 'loaded fswatch'
+augroup fswatch
```

Comment:
tbh, I'd also be OK with this moving to Lua, and just sourcing a `lua/fswatch.lua` file here with `luafile`, but I do not have a very strong preference on that.